dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -48,14 +48,27 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
         Multimap<String, String> pathToDockerfilesInParentRepo = ArrayListMultimap.create();
 
         Set<Map.Entry<String, JsonElement>> imageToTagStore = parseStoreToImagesMap(ns.get(Constants.STORE));
+        Integer gitApiSearchLimit = ns.get(Constants.GIT_API_SEARCH_LIMIT);
+        List<IOException> exceptions = new ArrayList<>();
+        Map<String, Boolean> orgsToIncludeInSearch = new HashMap<>();
```

Comment:
Are we wanting a `Set` instead? Should it be unmodifyable? What is the `value` here? Should this be a `Set` with typed `IncludedOrgs` / `ExcludedOrgs` or something like that?

Reply:
We wanted to have a map where the keys would be Org names and the value would be a Boolean value. True indicates the the org should be included in the search, and False indicates that it should not be included in the search. Using a map also ensures that the same org cannot be included and excluded at the same time.

Reply:
Gotcha. I guess I could read this a few ways. Since the variable is named `orgsToIncludeInSearch` I don't know that I'd immediately think of the include/exclude so that's where I was wondering if it deems a formal class for coherence. I guess the other way would be to add a comment for clarity.

Reply:
When the method call happens here, the map will only have an org to include in the search. Hence, I chose to name it `orgsToIncludeInSearch`. In the actual method definition, this gets stored in a variable `Map<String, Boolean> orgsToIncludeOrExclude` since this data structure can contain orgs to include or exclude depending on where it is being invoked from.